src/components/InviteToOpenSauced/InviteToOpenSaucedModal.ts

```diff
@@ -0,0 +1,47 @@
+import "../../index.css";
+import { TEST } from "../../config.json";
+
+interface Socials {
+  mailAddress?: string;
+  twitterUsername?: string;
+  linkedInUsername?: string;
+}
+
+export const InviteToOpenSaucedModal = (
+  username: string,
+  { mailAddress, twitterUsername, linkedInUsername }: Socials = {}
+) => {
+  //TODO: Conditionally render social icons
+  const socialIcons = `<span class="flex flex-nowrap space-x-3"> <a href="https://github.com/bdougie">
+    <img src="https://images.weserv.nl/?url=cdn4.iconfinder.com/data/icons/miu-flat-social/60/mail-512.png&h=60&w=60&fit=cover&mask=circle" alt="Email">
```

Comment:
Consider using SVGs (or an image) that can be bundled into the browser extension instead of relying on external assets.

Reply:
This was meant for prototyping the placement of the social icons. Will work on conditionally rendering them from local assets today.